test/entrypoint/config.yaml

```diff
@@ -29,6 +34,19 @@ groups:
                     format: prometheus
                     metrics:
                       enabled: true
+                - endpoints:
+                  - path: /metrics/prometheus
+                    format: prometheus
+                  - path: /metrics/json
+                    format: json
+                - server: 'additional'
+                  endpoints:
+                  - path: /metrics/prometheus
+                    format: prometheus
+                  - path: /metrics/json
+                    format: json
+                    metrics:
```

Comment:
Wait, what's these two lines are about?

Reply:
It is about this setting:

https://github.com/tarantool/metrics-export-role/blob/26923eebe7cb588c52f25b22b1bb2161a6438c7a/README.md?plain=1#L108-L118

But is is not that important in my example, so I removed it.

Reply:
I don't see any reason to remove it from an example.

Reply:
Name is really confusing, but ok

Reply:
> Oops, my mistake. Leaved example as it was and deleted `metrics.enabled` block from entrypoint config.

I think we need an integration test with httpd.server + an endpoint with metrics enable. Just to ensure that everything work together. So let’s return it to the test config.yaml for an endpoint and cover it by test.

Reply:
Done.